server/sql/prepend-i18n-bag-name.sql

```diff
@@ -0,0 +1,20 @@
+UPDATE bags b
+    JOIN user_chains bag_chain ON 
+        b.user_chain_id = bag_chain.id
+    JOIN user_chains uc ON
+        bag_chain.chain_id = uc.chain_id AND uc.is_chain_admin = 1
+    JOIN users u ON
+        uc.user_id = u.id AND LENGTH(u.i18n) = 2
```

Comment:
Correct me if I'm wrong but wouldn't this have the potential for it to be overwritten?

Loop x has two hosts A ( i18n: en ) & B ( i18n: empty )

The query finds bag K,and gives a list of;
1. Bag K, host A, en
2. Bag K, host B, empty

The SET would change the bag name first to "Bag K" then to "Tas K". 

If so use a sub query to find the first host with an actionable i18n (not "nl" or empty)

Reply:
Fixed (i hope). The query is still looping over all hosts, but will only change the number the first time. (order: nl, fr, en, others=nl)